pkg/destroy/libvirt/libvirt.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"github.com/openshift/installer/pkg/destroy/providers"
+	"github.com/openshift/installer/pkg/types"
```

Comment:
This seems not required..

Reply:
This does require because there it is been used in line number 50 in `New` function.